pkg/doc/verifiable/example_presentation_test.go

```diff
@@ -163,3 +167,290 @@ func ExampleCredential_Presentation() {
+			Name: "Example University",
+		},
+		Issued:  &issued,
+		Expired: &expired,
+		Schemas: []verifiable.TypedID{},
+		CustomFields: map[string]interface{}{
+			"referenceNumber": 83294847,
+		},
+	}
+
+	// The second VC is provided in JWS form (e.g. kept in the wallet in that form).
+	vcJWS := "eyJhbGciOiJFZERTQSIsImtpZCI6IiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE1Nzc5MDY2MDQsImlhdCI6MTI2MjM3MzgwNCwiaXNzIjoiZGlkOmV4YW1wbGU6NzZlMTJlYzcxMmViYzZmMWMyMjFlYmZlYjFmIiwianRpIjoiaHR0cDovL2V4YW1wbGUuZWR1L2NyZWRlbnRpYWxzLzE4NzIiLCJuYmYiOjEyNjIzNzM4MDQsInN1YiI6ImRpZDpleGFtcGxlOmViZmViMWY3MTJlYmM2ZjFjMjc2ZTEyZWMyMSIsInZjIjp7IkBjb250ZXh0IjpbImh0dHBzOi8vd3d3LnczLm9yZy8yMDE4L2NyZWRlbnRpYWxzL3YxIiwiaHR0cHM6Ly93d3cudzMub3JnLzIwMTgvY3JlZGVudGlhbHMvZXhhbXBsZXMvdjEiXSwiY3JlZGVudGlhbFNjaGVtYSI6W10sImNyZWRlbnRpYWxTdWJqZWN0Ijp7ImRlZ3JlZSI6eyJ0eXBlIjoiQmFjaGVsb3JEZWdyZWUiLCJ1bml2ZXJzaXR5IjoiTUlUIn0sImlkIjoiZGlkOmV4YW1wbGU6ZWJmZWIxZjcxMmViYzZmMWMyNzZlMTJlYzIxIiwibmFtZSI6IkpheWRlbiBEb2UiLCJzcG91c2UiOiJkaWQ6ZXhhbXBsZTpjMjc2ZTEyZWMyMWViZmViMWY3MTJlYmM2ZjEifSwiaXNzdWVyIjp7Im5hbWUiOiJFeGFtcGxlIFVuaXZlcnNpdHkifSwidHlwZSI6WyJWZXJpZmlhYmxlQ3JlZGVudGlhbCIsIlVuaXZlcnNpdHlEZWdyZWVDcmVkZW50aWFsIl19fQ.AHn2A2q5DL1heX3_izq_2yrsBDhoZ6BGGKhoRvhfMnMUuuOnBOdekdTg-dfUMJgipXRql_6WzBUIj4wTFehXCw" // nolint:lll
+
+	// Set the credentials.
+	err := vp.SetCredentials(vc, vcJWS)
+	if err != nil {
+		fmt.Println(fmt.Errorf("failed to set credentials of VP: %w", err))
+	}
+
+	//Output:
+}
+
+func ExamplePresentation_MarshalJSON() {
+	vp := &verifiable.Presentation{
+		Context: []string{
+			"https://www.w3.org/2018/credentials/v1"},
+		ID:     "urn:uuid:3978344f-8596-4c3a-a978-8fcaba3903c",
+		Type:   []string{"VerifiablePresentation"},
+		Holder: "did:example:ebfeb1f712ebc6f1c276e12ec21",
+	}
+
+	vc := verifiable.Credential{
+		Context: []string{
+			"https://www.w3.org/2018/credentials/v1",
+			"https://www.w3.org/2018/credentials/examples/v1"},
+		ID: "http://example.edu/credentials/1872",
+		Types: []string{
+			"VerifiableCredential",
+			"UniversityDegreeCredential"},
+		Subject: UniversityDegreeSubject{
+			ID:     "did:example:ebfeb1f712ebc6f1c276e12ec21",
+			Name:   "Jayden Doe",
+			Spouse: "did:example:c276e12ec21ebfeb1f712ebc6f1",
+			Degree: UniversityDegree{
+				Type:       "BachelorDegree",
+				University: "MIT",
+			},
+		},
+		Issuer: verifiable.Issuer{
+			ID:   "did:example:76e12ec712ebc6f1c221ebfeb1f",
+			Name: "Example University",
+		},
+		Issued:  &issued,
+		Expired: &expired,
+		Schemas: []verifiable.TypedID{},
+		CustomFields: map[string]interface{}{
+			"referenceNumber": 83294847,
+		},
+	}
+
+	err := vp.SetCredentials(vc)
+	if err != nil {
+		fmt.Println(fmt.Errorf("failed to set credentials of VP: %w", err))
+	}
+
+	// json.MarshalIndent() calls Presentation.MarshalJSON()
+	vpJSON, err := json.MarshalIndent(vp, "", "\t")
+	if err != nil {
+		fmt.Println(fmt.Errorf("failed to marshal VP to JSON: %w", err))
+	}
+
+	fmt.Println(string(vpJSON))
+
+	// Output: {
```

Comment:
Just want to confirm - is this comment the expected output? If so, it might be a little clearer to label it as "expected output" instead of just "output".

Reply:
It's a requirement of Golang examples - see [here](https://blog.golang.org/examples).